frontend/src/component/segments/SegmentFormStepTwo.tsx

```diff
@@ -38,6 +37,67 @@ interface ISegmentFormPartTwoProps {
     mode: SegmentFormMode;
 }
 
+const StyledForm = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+}));
+
+const StyledWarning = styled('div')(({ theme }) => ({
+    marginBottom: '1.5rem',
+}));
+
+const StyledInputDescription = styled('p')(({ theme }) => ({
+    marginBottom: '1rem',
+}));
+
+const StyledAddContextContainer = styled('div')(({ theme }) => ({
+    marginTop: '1rem',
+    borderBottom: `1px solid ${theme.palette.grey[300]}`,
+    paddingBottom: '2rem',
+}));
+
+const StyledError = styled('div')(({ theme }) => ({
+    marginTop: '1.5rem',
+}));
+
+const StyledNoConstraintText = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    marginTop: '6rem',
+}));
+
+const StyledSubtitle = styled('p')(({ theme }) => ({
+    fontSize: theme.fontSizes.bodySize,
+    color: theme.palette.grey[600],
```

Comment:
Should be using a semantic color from theme instead of leaking color specifics into the component.